src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/DependenciesGraphChangeTracker.cs

```diff
@@ -214,7 +216,7 @@ internal Enumerator(LinkedList<WeakReference<T>> list)
                 {
                     _list = list;
                     _next = list.First;
-                    Current = null;
+                    Current = null!;
```

Comment:
This was slightly interesting. In normal usage, the enumerator returns only non-null values. However before `MoveNext` is called, and after it returns `false`, `Current` is invalid and therefore null. It'd be technically more correct to make `Current` nullable, however all enumerations would then have to check every value for `null` which feels overkill given the enumerator works perfectly well with `foreach` as is.